src/components/Blog/index.css

```diff
@@ -0,0 +1,12 @@
+.blogContainer {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: nowrap;
+  /* align-items: center; */
+  justify-content: space-around;
+}
+
+/* .blogImg {
+  max-width: 100%;
+  height: 100%;
+} */
```

Comment:
If we don't need the code, it's better to delete them

Reply:
fixed